### PR TITLE
Migrate to subworkflow

### PR DESCRIPTION
**Pull request by CAMOBAP**
As title. 

 _Generated by Cimas_.

### CODE REVIEW

**Comment by ronaldtse** [comment]
@andrew2net @CAMOBAP is this to be merged? Thanks.

**Comment by ronaldtse** [comment]
@CAMOBAP will rebase this. Thanks!

**Comment by andrew2net** [comment]
The `.github/workflows/crawler.yml` has been changed. So it needs to update the `.github/workflows/crawler.yml`.

File: .github/workflows/crawler.yml [hunk 1]
```diff
@@ -1,34 +1,12 @@
+# Auto-generated by Cimas: Do not edit it manually!
+# See https://github.com/metanorma/cimas
 name: Crawler
 
 on:
   push:
   schedule:
-    - cron: '0 15 * * *'
-  workflow_dispatch:
```
Comment (andrew2net) [inline .github/workflows/crawler.yml:7]: Why has the workflow_dispatch been removed?

File: crawler.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+FileUtils.rm_rf('data')
+
+t1 = Time.now
+puts "Started at: #{t1}"
```
Comment (andrew2net) [inline crawler.rb]: It's already done the relaton-w3c. I don't think we need to put start/stop time here.

**Review by andrew2net — CHANGES_REQUESTED** [review]

File: crawler.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+FileUtils.rm_rf('data')
```
Comment (andrew2net) [inline crawler.rb:5]: We need to remove `index-w3c.yaml` as well.

File: crawler.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+t1 = Time.now
+puts "Started at: #{t1}"
+
+system("relaton fetch-data w3c-rdf")
```
Comment (andrew2net) [inline crawler.rb:10]: Before `relaton fetch-data w3c-rdf` we need to run `relaton fetch-data w3c-tr-archive`

File: .github/workflows/crawler.yml (hunk 1 above)
Comment (CAMOBAP) [reply .github/workflows/crawler.yml:7]: To be on the same page all relaton-data-* should have workflow_dipatch and cron?

Comment (andrew2net) [reply .github/workflows/crawler.yml:7]: Sometimes we need workflow_dispath to update files after changing in the data fetcher script, so we don't have to wait when cron runs it.

File: crawler.rb (hunk 2 above)
Comment (CAMOBAP) [reply crawler.rb]: Done

File: .github/workflows/crawler.yml (hunk 1 above)
Comment (CAMOBAP) [reply .github/workflows/crawler.yml:7]: added back

**Comment by CAMOBAP** [comment]
@andrew2net could you please have a look again

**Review by andrew2net — APPROVED** [review]

**Comment by CAMOBAP** [comment]
@andrew2net something is really bad happens( I will recreate PR

**Pull request closed without merging**